jobs/build/olm_bundle/Jenkinsfile

```diff
@@ -97,54 +82,31 @@ pipeline {
             description: 'Just show what would happen, without actually executing the steps',
             defaultValue: false,
         )
+        booleanParam(
+            name: 'MOCK',
+            description: 'Pick up changed job parameters and then exit',
+            defaultValue: false,
+        )
     }
 
     stages {
-        stage('Set build info') {
-            steps {
-                script {
-                    operator_nvrs = olm_bundles.commonlib.parseList(params.OPERATOR_NVRS)
-                    only = olm_bundles.commonlib.parseList(params.ONLY)
-                    exclude = olm_bundles.commonlib.parseList(params.EXCLUDE)
-                    currentBuild.displayName += " (${params.BUILD_VERSION})"
-
-                    if (params.ASSEMBLY && params.ASSEMBLY != "stream") {
-                        currentBuild.displayName += " - assembly ${params.ASSEMBLY}"
-                        if (params.EXTRAS_ADVISORY || params.METADATA_ADVISORY) {
-                            error("Cannot use EXTRAS_ADVISORY or METADATA_ADVISORY when building for non-stream assembly.")
-                        }
-                    } else {
-                        currentBuild.displayName  += " ${params.EXTRAS_ADVISORY ?: ''}"
-                    }
-                    if (operator_nvrs && (only || exclude)) {
-                        error("Cannot use OPERATOR_NVRS with ONLY or EXCLUDE.")
-                    }
-                    if (operator_nvrs && params.EXTRAS_ADVISORY) {
-                        error("Cannot use OPERATOR_NVRS with EXTRAS_ADVISORY.")
-                    }
-                    olm_bundles.buildlib.initialize(false, false)  // ensure kinit
```

Comment:
Don't see this as well, not needed?

Reply:
With EXTRAS_ADVISORY param being removed, this branch will never be (and currently never is) entered